core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -461,7 +464,7 @@ class KafkaController(val config: KafkaConfig, zkClient: KafkaZkClient, time: Ti
     // If replica failure did not require leader re-election, inform brokers of the offline replica
     // Note that during leader re-election, brokers update their metadata
     if (partitionsWithoutLeader.isEmpty) {
-      sendUpdateMetadataRequest(controllerContext.liveOrShuttingDownBrokerIds.toSeq)
+      sendUpdateMetadataRequest(controllerContext.liveOrShuttingDownBrokerIds.toSeq, Set.empty)
```

Comment:
Probably change the comment above from "inform brokers of the offline replica" to "inform brokers of the offline brokers"?

Reply:
Done.